web_src/less/_repository.less

```diff
@@ -881,6 +881,17 @@
                 }
 
                 .content {
+                    > .merge-section {
+                        .divider {
+                            margin-left: -1rem;
+                            margin-right: -1rem;
+                        }
+
+                        &.no-header {
+                            #avatar-arrow;
```

Comment:
What does this do? Copy the style?

Reply:
It's detached ruleset, I don't know why it was used initially but is used everywhere across this file to remove code duplication; it imports ::before rule for showing arrow.

Reply:
I see, the fact that these use `#` make them look like selectors. We should change them (in another PR) to use the `@` prefix so it's clearer, like the less docs do:

http://lesscss.org/features/#detached-rulesets-feature

Reply:
Agreed